client/state/selectors/get-reader-follows-count.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Get the count of follows a user has
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Integer} Follow count
+ */
+const getReaderFollowsCount = state => Math.max(
+	state.reader.follows.itemsCount,
+	Object.keys( state.reader.follows.items ).length,
```

Comment:
I'm not actually sure what to do here.....the count from the API can become out of date so we may not want to actually take the max between the two....